src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/AwakePage.xaml

```diff
@@ -132,57 +97,17 @@
                             MinWidth="90"
                             SmallChange="1" 
                             LargeChange="5"/>
+                    </StackPanel>
                 </StackPanel>
             </StackPanel>
-        </StackPanel>
-
-        <RelativePanel x:Name="SidePanel" 
-                    HorizontalAlignment="Left"
-                    Width="{StaticResource SidePanelWidth}"
-                    Grid.Column="1">
-            <StackPanel x:Name="DescriptionPanel">
-                <TextBlock x:Uid="About_Awake"
-                           x:Name="AboutTitle"
-                           Grid.ColumnSpan="2"
-                           Style="{StaticResource SettingsGroupTitleStyle}"
-                           Margin="{StaticResource XSmallBottomMargin}"/>
-                <TextBlock x:Uid="Awake_Description"
-                           TextWrapping="Wrap"
-                           Grid.Row="1" />
-            </StackPanel>
+        </controls:SettingsPageControl.ModuleContent>
 
-            <Border x:Name="AboutImage"
-                    CornerRadius="4"
-                    Grid.Row="2"
-                    MaxWidth="240"
-                    HorizontalAlignment="Left"
-                    Margin="{StaticResource SmallTopBottomMargin}"
-                    RelativePanel.Below="DescriptionPanel">
-                <HyperlinkButton x:Uid="Awake_ImageHyperlinkToDocs">
-                    <Image x:Uid="Awake_Image" Source="ms-appx:///Assets/Modules/Awake.png" />
-                </HyperlinkButton>
-            </Border>
-            <StackPanel x:Name="LinksPanel"
-                        Margin="0,1,0,0"
-                        RelativePanel.Below="AboutImage"
-                        Orientation="Vertical" >
-                <HyperlinkButton x:Uid="Awake_ImageHyperlinkToDocs">
-                    <TextBlock x:Uid="Module_overview" />
-                </HyperlinkButton>
-                <HyperlinkButton NavigateUri="https://aka.ms/powerToysGiveFeedback">
-                    <TextBlock x:Uid="Give_Feedback" />
-                </HyperlinkButton>
-
-                <TextBlock 
-                x:Uid="AttributionTitle"
-                Style="{StaticResource SettingsGroupTitleStyle}" />
-
-                <HyperlinkButton Margin="0,-3,0,0"
-                                 x:Uid="Awake_ModuleAttributionHyperlink">
-                    <TextBlock x:Uid="Awake_ModuleAttributionLabel"
-                               TextWrapping="Wrap" />
-                </HyperlinkButton>
-            </StackPanel>
-        </RelativePanel>
-    </Grid>
-</Page>
+        <controls:SettingsPageControl.ModuleLinks>
+            <controls:SidePanelLink x:Uid="Learn_More" Link="https://aka.ms/PowerToysOverview_Awake"/>
+            <controls:SidePanelLink x:Uid="Give_Feedback" Link="https://aka.ms/powerToysGiveFeedback"/>
```

Comment:
just an idea.. as Give feedback link is always present and the same, might be worth considering defaulting it in SidePanelLink

Reply:
Yeah, I thought about that.. for flexibility reasons I thought this would be better. But you raise a good point, as the link is the same everywhere and links to the generic issues page of the repo it's not super useful. Let's leave it for now.